subprojects/plugins/src/main/java/org/gradle/api/plugins/internal/JvmPluginsHelper.java

```diff
@@ -274,16 +273,6 @@ public static void configureAttributesForRuntimeClasspath(ConfigurationInternal
         };
     }
 
-    private static int getReleaseOption(List<String> compilerArgs) {
```

Comment:
:o: I know we discussed this back and forth and I did not want to add this in the first place. But now that we have this (and it was released), should we maybe keep this for now (at least until the new property is not longer Incubating)? If we remove it now, it might break publishing for users that started to rely on it with the last release.

Reply:
Good point, I missed the exact meaning of this when merging. I'll add it back with the proper testing.